cmd/user/internal/interfaces/http/router.go

```diff
@@ -39,6 +40,7 @@ func NewRouter(logger *log.Logger, repository user_persistence.UserRepository, c
 		http_middleware.HSTS(),
 		http_middleware.Metrics(),
 		http_middleware.LimitRequestBody(int64(10<<20)), // 10 MB is a lot of text.
+		http_form_middleware.FormJson(),
```

Comment:
in user service we have limit request body after formJson 
and here its the other way around

I think we should be consistent right ?

Reply:
https://github.com/vardius/go-api-boilerplate/pull/60/files#diff-13f225e09e5167485fe2fb11c089358aR29-R30

Reply:
in fact I think http_middleware.LimitRequestBody should be before http_form_middleware.FormJson

so you have to update user service